index.html

```diff
@@ -0,0 +1,18 @@
+<!-- --﷽-- -->
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Linter</title>
+    <link
+      href="http://fonts.googleapis.com/css?family=Open+Sans"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+</html>
+
+
+    <h1> Hello Microverse!</h1>
```

Comment:
I suggest that you  Use a proper document structure, put all your tags inside a body tag withing your html tag, it will work in some browsers like Chrome and Firefox without problems but it will break in others. Generally, it's not recommended to do that because, as I already said, it might break functionality in some browsers

Reply:
Resolved....Thanks for the feedback